### ISSUE TITLE
SPARQL: Add DataDistribution queries for analysis

### ISSUE THREAD

**Issue by valentinedwv** [earthcube/earthcube_utilities#92]
I think we need to have an idea on what can be downloaded

How many JSON-LD Datasets have url's 

distribution:
* @type and counts (are there any besides dataDownload)
* encodingFormat and name

Service Endpoints

**Comment by lmarini** [comment]
We want to do it in the graph so that we can do internal analysis and then show it on the web app as summaries.

**Comment by valentinedwv** [comment]
> We want to do it in the graph so that we can do internal analysis and then show it on the web app as summaries.

Yes. We want to do it in spaql, or over a Release file using RDF lib so we can create a report.

might even be a some accessibility score to display in the UI... this is a landing page, this is an actual dataset, this is many datasets.

**Comment by ylyangtw** [comment]
some questoions like: how many distributions

**Comment by ylyangtw** [comment]
Need to show duplicates. Deep Oceans may need this.

**Comment by valentinedwv** [comment]
cache or some example sparql https://github.com/earthcubearchitecture-project418/assay-data/blob/master/README.md#dataset-distribution

**Comment by valentinedwv** [comment]
while you are looking and learning a bit of sparql...

**Comment by ylyangtw** [comment]
@ywkim312 You can try playing SPARQL queries in this graph https://graph.geocodes.ncsa.illinois.edu/blazegraph/#namespaces
Use the namespace `decoder` and try SPARQL query in the tab `Query`

Also, Qlever: https://qlever-ui.geocodes-aws-dev.earthcube.org/

Example queries:
```
SELECT *
WHERE {
  ?s ?p ?o .
}
LIMIT 200
```

All the SPARQL queries in this repo: https://github.com/earthcube/earthcube_utilities/tree/main/earthcube_utilities/src/ec/graph/sparql_files
